static/admin/js/admin.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+  'use strict';
+  //for the tests window isn't the global object
+  var $ = window.$;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['$' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/982033828/issues/source?bid=2452379&fileBranchId=2865970#l4)